build/changelog/entries/2015/12/10431.SUP-2214.bugfix

```diff
@@ -0,0 +1 @@
+Drag and drop of content inside of an editable was handled by the browser instead of the Aloha Editor and caused problems. For example, when dragging and dropping a link inside of an editable, the link text got underlined. Now drag and drop is blocked but it still works for blocks as intendet.
```

Comment:
Native drag and drop handling by the browser causes unwanted style elements to be added around text. Drag and drop of text is now disabled.